release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3326,6 +3326,62 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-34_{context}"]
+=== RHSA-2024:4960 - {product-title} 4.14.34 bug fix and security update
+
+Issued: 08 August 2024
+
+{product-title} release 4.14.34, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4960[RHSA-2024:4960] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4963[RHSA-2024:4963] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.34 --pullspecs
+----
+
+[id="ocp-4-14-34-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-14-34-timeout-ingress_{context}"]
+===== Adding connectTimeout tuning option to Ingress Controller API
+
+* The IngressController API is updated with a new tuning option, `ingresscontroller.spec.tuningOptions.connectTimeout`, which defines how long the router will wait for a response when establishing a connection to a backend server. (link:https://issues.redhat.com/browse/OCPBUGS-36555[*OCPBUGS-36555*])
```

Comment:
Suggestions to avoid future tense and hyphenate "back-end" when it's an adjective, per the ISG

```suggestion
* The IngressController API is updated with a new tuning option, `ingresscontroller.spec.tuningOptions.connectTimeout`, which defines how long the router waits for a response when establishing a connection to a back-end server. (link:https://issues.redhat.com/browse/OCPBUGS-36555[*OCPBUGS-36555*])
```